collector/pkg/component/analyzer/network/protocol/testdata/na-protocol-config.yaml

```diff
@@ -26,3 +26,6 @@ analyzers:
       - key: "dns"
         ports: [ 53 ]
         slow_threshold: 100
+      - key: "rocketmq"
+        ports: [ 9876 ]
```

Comment:
Do not set specified port in testCase yaml. Make sure the rocketmq payload not identifed as other protocols.